modules/RTC/RTCUtils.js

```diff
@@ -668,6 +668,8 @@ function defaultGetVideoSrc(element) {
         if (URL) {
             url = URL.createObjectURL(srcObject);
             try {
+                // store the url to avoid creating different url for every call
+                srcObject.jitsiObjectURL = url;
```

Comment:
There's a URL.revokeObjectURL(url) which disposes of the value assigned to srcObject.jitsiObjectURL!